src/arithmetic/path_funcs/path_funcs.c

```
@@ -92,169 +96,189 @@
 	return path;
 }
 
-// Routine for freeing a shortest path function's private data.
-void ShortestPath_Free(void *ctx_ptr) {
-	ShortestPathCtx *ctx = ctx_ptr;
-	if(ctx->reltypes) array_free(ctx->reltypes);
+// routine for freeing a shortest path function's private data
+void ShortestPath_Free
+(
```

Cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)